contracts/staking/TokenStaking.sol

```diff
@@ -328,6 +329,36 @@ contract TokenStaking is Initializable, IStaking, Checkpoints {
         emit ApplicationStatusChanged(application, ApplicationStatus.APPROVED);
     }
 
+    /// @notice Update `authorizedOverall` field for each application.
+    ///         Can only be called by Governance. `authorizedOVerall` should be
+    ///         equal to the sum of all authorization for particular application.
+    // TODO remove after use
+    function updateAuthorizedOverall(int96[] memory authorizedOverallValues)
+        external
+        onlyGovernance
+    {
+        require(
+            authorizedOverallValues.length == applications.length,
+            "Wrong input parameters"
+        );
+        for (uint256 i = 0; i < applications.length; i++) {
+            address application = applications[i];
+            ApplicationInfo storage applicationStruct = applicationInfo[
+                application
+            ];
+            int96 authorizedOverall = authorizedOverallValues[i];
+            if (authorizedOverall >= 0) {
+                applicationStruct.authorizedOverall += uint96(
```

Comment:
Why not directly setting `applicationStruct.authorizedOverall`, rather than using `int96` increments/decrements?

Reply:
because with just setting value we always can face frontrun tx that will change authorization and now we have to execute one more tx. With addition or subtraction - maximum we need two txs even if one/many will be executed right before upgrade